app/src/components/Nav.css

```diff
@@ -0,0 +1,27 @@
+.navbar{
+  display: flex;
+  width: 100%;
+  background: #1F2630;
+  justify-content:space-between;
+}
+
+.navbar h3{  
+  margin-left: 3rem;
+  font-family: 'Arial';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 30px;
+  line-height: 34px;
+  color: aliceblue;
+}
+
+@media (max-width: 475px) {
+  .navbar h3{  
+    margin-left: 1.5rem;
+  }
+
+  .button {
+    margin: auto 0.5rem;
+    width: 120px;
```

Comment:
rem instead of pixel ideally, though with media queries maybe u can get away with this....@Ebrahim-Merchant watcha say ?

Reply:
if you use rem... you can just change the `root element module - rem` which should be the `body` tag stylings pixels to change everything accordingly to the rem change. So the way rem works is its just a multiple of the root module which is usually 16px by default.